jsconcat.php

```diff
@@ -84,6 +84,12 @@ function do_items( $handles = false, $group = false ) {
 			else
 				$js_url['path'] = substr( $js_realpath, strlen( ABSPATH ) - 1 );
 
+			//Don't concat items with associated inline scripts
+			$before_handle = $this->print_inline_script( $handle, 'before', false );
+			$after_handle = $this->print_inline_script( $handle, 'after', false );
+			if ( $before_handle || $after_handle )
```

Comment:
Let's use brackets here, per the WP coding standards [#](https://make.wordpress.org/core/handbook/best-practices/coding-standards/php/#brace-style)